provider/postgis/postgis_test.go

```diff
@@ -68,12 +68,8 @@ func TestMVTLayer(t *testing.T) {
 					},
 				},
 			},
-			tile: tegola.Tile{
-				Z: 1,
-				X: 1,
-				Y: 1,
-			},
-			expectedFeatureCount: 614,
+			tile:                 tegola.NewTile(1, 1, 1),
+			expectedFeatureCount: 4032,
```

Comment:
Did you test the new feature counts? Seems like a big jump by just adding the tile buffer.

Reply:
Yes, this is from the test db.